Sources/SwiftSentry/SentrySDK.swift

```diff
@@ -255,6 +260,13 @@ public enum SentrySDK {
             let hresult = String(UInt32(bitPattern: stowedException.resultCode), radix: 16)
             let exception = sentry_value_new_exception("StowedException", "Stowed exception #\(index + 1) - HRESULT: 0x\(hresult)")
             sentry_value_set_stacktrace(exception, ips, Int(stowedException.stackTraceCount))
+
+            if isMostRecent {
+              let mechanism = sentry_value_new_object()
```

Comment:
If you don't set the mechanism it shows as `generic`. Should you report it as `stowed`?

Reply:
The mechanism aren't documented and I wanted to avoid using some random values here, I'm also not sure if using "stowed" will change much in practice?

Reply:
I dunno, we could use it as a filter, but not a big deal.